web/src/test/resources/com/epam/rft/atsy/cucumber/Welcome.feature

```diff
@@ -2,6 +2,7 @@ Feature: Welcome
 
   Background:
     Given The user signed in
+    Given The Applications of the candidate with id 1 are deleted
```

Comment:
If you come back to this feature file after few months and you see this line, then you will have no idea which is the candidate with id 1.

I suggest a more readable sentence here, like:

    Given The following applications for positions exist:
        | candidate   | position  |
        | Candidate A |           |
        | Candidate B |           |
        | Candidate C | Fejlesztő |

In this case your java code can make sure that the A and B candidates have no applications, but candidate C has one with position `Fejlesztő`.
I suggest to rework the temporary endpoint to support this functionality.